include/manager.hpp

```diff
@@ -30,15 +31,85 @@ class Manager
      * @param[in] iFace - interface to implement.
      * @param[in] connection - Dbus Connection.
      */
-    Manager(const std::shared_ptr<boost::asio::io_context>& ioCon,
-            const std::shared_ptr<sdbusplus::asio::dbus_interface>& iFace,
-            const std::shared_ptr<sdbusplus::asio::connection>& asioConnection);
+    Manager(
+        const std::shared_ptr<boost::asio::io_context>& ioCon,
+        const std::shared_ptr<sdbusplus::asio::dbus_interface>& operationIFace,
+        const std::shared_ptr<sdbusplus::asio::connection>& asioConnection);
 
     /**
      * @brief Destructor.
      */
     ~Manager() = default;
 
+    /**
+     * @brief Write a keyword value on the given placeholder.
```

Comment:
Team:
api name : updateKeyword();
Rewordings: Update keyword value.
add examples: 
Design: for both : input should be dbus object path. 
Path - (dbus path or eeprom path)

<applicable for both read and write>